core/controllers/questions_list.py

```diff
@@ -63,10 +68,27 @@ def get(self, comma_separated_skill_ids):
 
         (
             question_summaries, merged_question_skill_links,
-            next_start_cursor) = (
-                question_services.get_displayable_question_skill_link_details(
-                    constants.NUM_QUESTIONS_PER_PAGE, skill_ids, start_cursor)
-            )
+            next_offset
+        ) = (
+
+            question_services.get_displayable_question_skill_link_details(
+                constants.NUM_QUESTIONS_PER_PAGE, skill_ids, offset=offset)
+        )
+
+        _, _, temp_next_offset = (
+            question_services.get_displayable_question_skill_link_details(
+                constants.NUM_QUESTIONS_PER_PAGE, skill_ids,
+                offset=next_offset)
+        )
+
+        # To check whether there are more questions.
+        if next_offset == temp_next_offset:
+            more = False
+
+        # Pass null to frontend if there are no more questions.
+        if not more:
```

Comment:
I don't really understand why you need "more" anyway? You could just do `if next_offset == temp_next_offset` here, right (unless I'm misunderstanding/misreading)?

Reply:
Done